src/components/_basic.less

```diff
@@ -72,3 +72,12 @@ a.sbtn {
 .purple-btn {
   .colorRule(@purple);
 }
+
+.white-btn {
+  .colorRule(@lightText);
```

Comment:
Can you please create a variable for white color? I know it's the same but it would be better in case of any future changes and it would be clearer.

Reply:
variable added for white color! thank you!